client/plots/matrix.controls.js

```diff
@@ -1111,21 +1111,41 @@ export class MatrixControls {
 
 	generateCNVItems(self, app, parent, table) {
 		table.attr('class', null) // remove the hoverover background for CNV button
+		const m = parent.config.settings.matrix
+		const mutationLegendGrp = parent.legendData.find(l => l.dt?.includes(1))
```

Comment:
let's use `dtsnvindel` instead of 1. please change all such uses in matrix. that can make code impossible to maintain